src/main/java/mate/academy/spring/model/User.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
User is Component?

Reply:
Why not?
I redid after reading [this articl](https://javarush.ru/groups/posts/477-spring-dlja-lenivihkh-osnovih-bazovihe-koncepcii-i-primerih-s-kodom-chastjh-2).
![image](https://user-images.githubusercontent.com/57962724/140268477-46b6bb40-2082-417c-bebd-5bf95a2d8f7e.png)